src/main/webapp/WEB-INF/views/manufacturers/create.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
```

Comment:
Title -?

Reply:
Sorry. Changed it in index and here we have another one. ;)